TPAC-2020/readme.md

```diff
@@ -76,6 +74,8 @@ IRC: [irc.w3.org](http://irc.w3.org/?channels=#immersive-web) #immersive-web [In
                 <td rowspan="6">
                     <ul>
                         <li>Open for Unconference sessions, demos and lightining talks <b>10:00 AM - 1:00 PM PST</b>
```

Comment:
end should be 12:00, but not 1:00 PM (due to XR accessibility has moved to Thu)?